src/main/java/io/stargate/sgv2/jsonapi/service/cqldriver/executor/NamespaceCache.java

```diff
@@ -58,6 +68,8 @@ protected Uni<CollectionSettings> getCollectionProperties(
                                     .concat(collectionName)));
                   }
                   // collection does not exist
+                  // TODO: DO NOT do a string starts with , use property error structures
+                  // again, why is this here, looks like it returns the same error code ?
```

Comment:
That is odd also since we for some reason do not check if it's `JsonApiException` (which is a `RuntimeException`)...